### PR TITLE
Github Actions master builds

### PR DESCRIPTION
**Pull request by jherico**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22748/HighFidelity-Beta-PR16552-f39088f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22748/HighFidelity-Beta-Interface-PR16552-f39088f.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22748/HighFidelity-Beta-PR16552-f39088f.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22748/HighFidelity-Quest-Beta-PR16552-f39088f.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22749/HighFidelity-Beta-PR16552-8844620.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22749/HighFidelity-Beta-Interface-PR16552-8844620.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22749/HighFidelity-Beta-PR16552-8844620.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22749/HighFidelity-Quest-Beta-PR16552-8844620.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22749/HighFidelity-Beta-PR16552-8844620.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22749/HighFidelity-Beta-Interface-PR16552-8844620.exe

File: CMakeLists.txt [hunk 1]
```diff
@@ -257,6 +257,11 @@ if (ANDROID)
   return()
 endif()
 
+if (BUILD_GPU_FRAME_PLAYER_ONLY)
```
Comment (hd23408) [inline CMakeLists.txt]: Did you want to leave this in for future testing, or remove?

Comment (jherico) [reply CMakeLists.txt]: I'd like to leave it in.  It's a pain to have to keep modifying the cmake files whenever I want to reduce the build to 5 minutes.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22760/HighFidelity-Beta-PR16552-d311b2d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22760/HighFidelity-Beta-Interface-PR16552-d311b2d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22760/HighFidelity-Beta-PR16552-d311b2d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22760/HighFidelity-Beta-Interface-PR16552-d311b2d.exe

File: .github/workflows/master_build.yml [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+      with: 
+        submodules: true
+        fetch-depth: 1
+    - name: Create Build Directory
+      run: cmake -E make_directory ${{runner.workspace}}/build
+    - name: Decrypt Signing Key (Windows)
+      if: matrix.os == 'windows-latest'
+      working-directory: ${{runner.workspace}}/build
+      shell: bash
+      run: gpg --batch --yes -o codesign.pfx --passphrase "${{secrets.gpg_symmetric_key}}" --decrypt $GITHUB_WORKSPACE/tools/ci-scripts/codesign.pfx.gpg
```
Comment (AndrewMeadows) [inline .github/workflows/master_build.yml]: Is indentation important in these YML config files?  It looks to me like lines 171-173 should be indented one level.  Also lines 176-181(?),  and several other instances below.  If this config is working then I assume the indentation is only useful for human readability, and I would guess... the context of an `if` block extends to the next `-` bullet point.

Comment (jherico) [reply .github/workflows/master_build.yml]: the `if` conditional here specifies whether the job named at line 169 should run.  The individual jobs are delimited by the `-` markers.

**Pull request merged**